drivers/led/Kconfig.lp503x

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2018 Linaro Limited
```

Comment:
You should have your employers copyright here also!

Reply:
Well spotted. Fixed.